src/unraphael/dash/align.py

```diff
@@ -466,3 +469,158 @@ def align_image_to_base(
         raise ValueError(f'No such method: {align_method}')
 
     return func(image=image, base_image=base_image, **kwargs)
+
+
+def feature_based_alignment_visual(
+    base_image: np.ndarray,
```

Comment:
I suggest protecting this function by requiring keyword arguments:

```suggestion
def feature_based_alignment_visual(
    *,
    base_image: np.ndarray,
```